clippy_lints/src/methods/mod.rs

```diff
@@ -2185,6 +2193,22 @@ fn lint_map_or_none<'a, 'tcx>(cx: &LateContext<'a, 'tcx>, expr: &'tcx hir::Expr,
     }
 }
 
+fn lint_filetype_is_file(cx: &LateContext<'_, '_>, expr: &hir::Expr, args: &[hir::Expr]) {
+    const LINT_MSG: &str = "`FileType.is_file()` does not cover all readable file types in linux.";
```

Comment:
What about other platforms? Clippy has to support Linux, macOS and Windows.

```suggestion
    const LINT_MSG: &str = "`FileType::is_file()` does not cover all readable file types in linux";
```